bookshelves.md

```diff
@@ -8,39 +8,60 @@ The Free Ebook Foundation (FEF) has been helping Project Gutenberg (PG) renovate
 
 “Bookshelves” are lists of PG books that help users navigate PG. See them [here](https://www.gutenberg.org/wiki/Category:Bookshelf).
 
-
 ## Problem
 
 - System for creating and updating bookshelves is failing
-    - PHP/wiki interface
-    - Loaded into production Postgres db by scraping wiki!
-    - Few people maintaining it.
-    - No way to create personal bookshelves or reading lists
-    - Recurring security issues with PHP
-
+  - PHP/wiki interface
+  - Loaded into production Postgres db by scraping wiki!
+  - Few people maintaining it.
+  - No way to create personal bookshelves or reading lists
+  - Recurring security issues with PHP
 
 ## Proposed Solution
 
 - Registrationless Bookshelf Builder
-    - HTML5 local storage + javascript
-    - Button on Item page
-    - Import/export lists for sharing
-
-- Bookshelf Management App
-    - Python/Flask or Python/CherryPy with Postgres backend
-    - Managers can make new bookshelves and categories
-    - Moderators can add/remove items from Builder Bookshelves
 
+  - HTML5 local storage + javascript
+  - Button on Item page
+  - Import/export lists for sharing
 
+- Bookshelf Management App
+  - Python/Flask or Python/CherryPy with Postgres backend
+  - Managers can make new bookshelves and categories
+  - Moderators can add/remove items from Builder Bookshelves
 
 ## Goal
 
 Have prototype system running by April, 2021 for public review, move to production bin May.
 
+## Timeline
+
+| Dates               |                                                               Milestones                                                               |
+| ------------------- | :------------------------------------------------------------------------------------------------------------------------------------: |
+| 09/14/20 - 9/21/20  | Review Technical Specs, existing code/documentation, and implementation goals for deeper understanding of technical aspects of problem |
+| 09/25/20            |   Deliver implementation plan and user stories for dev team to iteratively implement; allowing the dev team to reevaluate any cases.   |
+| 09/28/20 - 04/30/21 |    Main Development Period: Iteratively develop user stories several sprints. Demo new features at the end of sprints for feedback.    |
+| 04/30/21            |                                                       Final demo of new features                                                       |
+| 04/30/21 - 05/14/21 |                              QA period to resolve issues and bugs. Final preparation for product release                               |
+
+_Dates/Milestones are tentative and subject to change._
+
 ## Team
 
-Add names and links here.
+Adam Undus (aundus@stevens.edu)
+Liam Nagel (nagel@stevens.edu)
+John Dyer (jdyer@stevens.edu)
+Damon Del Priore (ddelprio@stevens.edu)
+Anthony Ciccone (aciccone@stevens.edu)
+
+## More about our team
```

Comment:
Do any of you have python experience?

Reply:
Yes, I believe all of us have some Python experience.